puppet/modules/quickstack/manifests/pacemaker/neutron.pp

```diff
@@ -250,22 +250,16 @@
       operation_opts   => "start timeout=90",
       # monitor_params => { 'start-delay'     => '10s' },
     }
-    ->
-    quickstack::pacemaker::resource::generic {'neutron-scale':
-      resource_name => 'neutron:NeutronScale',
-      clone_opts    => "globally-unique=true clone-max=${_clone_max} interleave=true",
-      resource_type => 'ocf',
-    }
 
     if ($core_plugin == 'plumgrid') {
-      notify { 'PLUMgrid neutron-scale resource':
+      notify { 'PLUMgrid core plugin':
         require => Quickstack::Pacemaker::Resource::Generic['neutron-server'],
       }
       ->
       Anchor['pacemaker ordering constraints begin']
 
     } else {
-      notify { 'neutron-scale resource':
+      notify { 'not PLUMgrid core plugin':
```

Comment:
I'm not saying that is a bad thing necessarily, but that feels like more of a refactor and out of scope of this PR.  The notify's are both a little awkward to begin with IMO, but still I'd rather not touch them here.